tvtk/tvtk_base.py

```diff
@@ -16,23 +16,10 @@
 from traits import api as traits
 from . import messenger
 
-try:
```

Comment:
Hmm. I removed this `try / except` because I thought it wasn't needed any more.  But we currently explicitly declare only that Mayavi requires Traits >= 6.0, and `PrefixMap` and `PrefixList` were introduced in Traits 6.1. I think we should probably update the minimum Traits version required to Traits 6.1.

Reply:
Done.